### PR TITLE
Oculus via Unity OpenXR

### PR DESCRIPTION
**Pull request by merulaxr**
Switches Oculus to build via Unity's OpenXR rather than Oculus's custom plugin 🎉

- Removes Oculus' plugin from the package manager.
- update to v39 of the Oculus SDK to enable the new OpenXR Feature.
- new IDisposable for managing OpenXR features.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -231,14 +231,6 @@ jobs:
           restore-keys: |
             Library-${{ matrix.targetPlatform }}-
 
-      - name: Manually delete OpenXR packages
```
Comment (mikeage) [inline .github/workflows/build.yml:234]: So glad to see this gone! :-)

File: Assets/Editor/BuildTiltBrush.cs [hunk 2]
```diff
@@ -859,6 +857,11 @@ static void CommandLine()
             target = BuildTarget.StandaloneWindows64;
         }
 
+        if (target == BuildTarget.Android)
+        {
+            EditorUserBuildSettings.androidCreateSymbols = AndroidCreateSymbols.Debugging;
```
Comment (mikeage) [inline Assets/Editor/BuildTiltBrush.cs:862]: I think we always want this set, even if it's not a debug build. It doesn't change anything about the executable files, just the (extra) symbols.zip

**Review by mikeage — CHANGES_REQUESTED** [review]
One comment about the symbols. Otherwise looks good, assuming it works :-)

(unfortunately, the builds take caches from their branches and/or from `main` , but not from the target branch, so it did have to build everything)

File: Assets/Editor/BuildTiltBrush.cs (hunk 2 above)
Comment (merulaxr) [reply Assets/Editor/BuildTiltBrush.cs:862]: This was moved from elsewhere in the file where it was previously only set for Oculus platforms, I think it's always generated regardless of debug or not!

Comment (mikeage) [reply Assets/Editor/BuildTiltBrush.cs:862]: Right, and that's good! But we used to set it to true, and now it's only set if Debugging is enabled.

But there's a simple test; is it generated now? If so, let's merge. If not, let's go back to just "true".

I'll check tomorrow am, if you don't get to it first.

Comment (merulaxr) [reply Assets/Editor/BuildTiltBrush.cs:862]: the `true` method is deprecated in 2021, now you can choose either `Public` or `Debugging`, of which I assume Debugging is what we want! https://docs.unity3d.com/2021.2/Documentation/ScriptReference/AndroidCreateSymbols.html

Comment (mikeage) [reply Assets/Editor/BuildTiltBrush.cs:862]: oh, ok, I totally misunderstood this! Never mind :-)

**Review by mikeage — APPROVED** [review]

**Comment by merulaxr** [comment]
I've put this on draft while I investigate how Oculus layers work, and if OB is using them incorrectly. Technically not a blocker but may have to revert this if it blocks other Oculus features.

**Comment by billyquith** [comment]
Good to see the Gradle conflict issue being resolved (and without a custom Gradle, which just makes things more complicated). 

So we still need the Oculus SDK for the extensions I guess, which in time will migrate into the plugin and then at some stage it can be removed?

**Comment by merulaxr** [comment]
Yep, and fingers crossed is goes into the package instead!

**Pull request merged**